HLTriggerOffline/JetMET/python/Validation/SingleJetValidation_cfi.py

```diff
@@ -17,7 +17,8 @@
 
 from Configuration.Eras.Modifier_phase2_common_cff import phase2_common
 phase2_common.toModify(SingleJetMetPaths,
-                       PatternJetTrg = "HLT_(AK4)?PFPuppiJet([0-9])+(_v[0-9]+)?$",
+                       PatternJetTrg  = "HLT_(AK4)?PFPuppiJet([0-9])+(_v[0-9]+)?$",
+                       PatternMetTrg  = "HLT_PFPuppiMET+[TypeOne140_PFPuppiMHT140]+(_v[0-9]+)?$",
```

Comment:
@missirol @SohamBhattacharya Could you please advise on string for MET path? Or ping responsible person.

I think I will stop after this PR as we should be ready to handle Phase-2 HLT DQM. And let Phase-2 HLT developers to handles all paths.

Thanks.